code/modules/mob/living/silicon/mommi/mommi.dm

```diff
@@ -40,6 +40,12 @@ They can only use one tool at a time, they can't choose modules, and they have 1
 /mob/living/carbon/can_use_hands()
 	return 1
 
+/mob/living/silicon/robot/mommi/getFeetAmount()
+	return 4
+
+/mob/living/silicon/robot/mommi/getHandAmount()
+	return 2
```

Comment:
The implementation has one, and the original crab sprite had a tool arm and a claw on the other.